bin/config.js

```diff
@@ -1,19 +1,11 @@
-const rc = require('rc')
+const fsproxy = require('fs-proxy')
 
-module.exports = rc('cordlr', {
```

Comment:
Maybe using `fsproxy` is too limiting with our goals in mind, i think im going to (slightly) rewrite the `config.js` file to just read once. Shoud i include a write method in the module.exports to write the config to the file?